app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class = "row"> 
+    <div class ="col-3">
+        <div class ="card mt-4">
+            <div class ="card-body">
+                <ul class="sidebar-list">
+                    <li class="sidebar-item">
+                        <%= link_to "Your Listings", rooms_path, class: "sidebar-link active" %>
+                    </li>
+                    <li class="sidebar-item">
+                        <%= link_to "Your Reservations", "#", class: "sidebar-link active" %>
+                    </li>
+                    <li class="sidebar-item">
+                        <%= link_to "Your Trips", "#", class: "sidebar-link active" %>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+
+    <div class = "col-9">
+        <div class = "card mt-4">
+            <div class = "card-header text-white">
+                Your Listings 
+            </div>    
+
+            <div class="card-body">
+                <div class = "container ">
+                    <div class = "row">
+                        <% @rooms.each do |room| %>
```

Comment:
You can use current_user.rooms here. current_user is a devise helper that returns a current logged in user.